packages/components/src/external-link/index.tsx

```diff
@@ -54,28 +62,33 @@ function UnforwardedExternalLink(
 
 	return (
 		/* eslint-disable react/jsx-no-target-blank */
-		<a
-			{ ...additionalProps }
-			className={ classes }
-			href={ href }
-			onClick={ onClickHandler }
-			target="_blank"
-			rel={ optimizedRel }
-			ref={ ref }
-		>
-			<span className="components-external-link__contents">
-				{ children }
-			</span>
-			<span
-				className="components-external-link__icon"
-				aria-label={
-					/* translators: accessibility text */
-					__( '(opens in a new tab)' )
-				}
-			>
-				&#8599;
-			</span>
-		</a>
+		<Fragment>
+			<div className="components-external-link__wrapper">
+				<a
+					{ ...additionalProps }
+					className={ classes }
+					href={ href }
+					onClick={ onClickHandler }
+					target="_blank"
+					rel={ optimizedRel }
+					ref={ ref }
+				>
+					<span className="components-external-link__contents">
+						{ children }
+					</span>
+					<span
+						className="components-external-link__icon"
+						aria-label={
+							/* translators: accessibility text */
+							__( '(opens in a new tab)' )
+						}
+					>
+						&#8599;
+					</span>
+				</a>
+				<div>{ suffix && <Suffix>{ suffix }</Suffix> }</div>
```

Comment:
is the `<div>` necessary?

Reply:
Yes, as I'm using `space-between` in styling so it was needed